PulleyLib/PulleyViewController.swift

```diff
@@ -298,9 +298,18 @@ open class PulleyViewController: UIViewController, PulleyDrawerViewControllerDel
             return drawerScrollView.bounds.height
         }
     }
-    
+
+    // Returns default blur style depends on iOS version.
+    private static var defaultBlurEffect: UIBlurEffect.Style {
+        if #available(iOS 13, *) {
+            return .systemUltraThinMaterial
```

Comment:
ultra thin might be too transparent for some use cases. Does it make sense to add another `PulleyDrawerViewControllerDelegate` property so that consumer can customize this if necessary?